docs/release_notes/v0.11.0.md

```diff
@@ -0,0 +1,489 @@
+
+```bash
+dapr uninstall --all
+```
+
+Next, get the latest CLI following [these](https://github.com/dapr/cli#installing-dapr-cli) instructions, or alternatively download the latest and greatest release from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+Once you have downloaded the CLI, run:
+
+```bash
+dapr init
+```
+
+Wait for the update to finish, and you're good to go!
+Make sure you have the right runtime version (0.11.0) with:
+
+```bash
+dapr --version
+```
+
+### Kubernetes
+
+#### Upgrading from 0.10.0
+
+If you previously installed Dapr using Helm, starting with this releases you can now upgrade Dapr to a new version.
+If you installed Dapr using the CLI, go [here](#starting-fresh).
+
+##### 1. Get the latest CLI
+
+Get the latest version of the Dapr CLI as outlined above, and put it in your path.
+You can also use the helper scripts outlined [here](https://github.com/dapr/cli#installing-dapr-cli) to get the latest version.
+
+##### 2. Upgrade
+
+First, update your Helm repos:
+
+```
+helm repo update
+```
+
+Run the following command to upgrade Dapr:
+
+```
+helm upgrade dapr dapr/dapr --version 0.11.0 --namespace dapr-system --reset-values
+```
+
+Wait until all the pods are in Running state:
+
+```
+kubectl get pods -w -n dapr-system
+```
+
+Verify the control plane is updated and healthy:
+
+```
+dapr status -k
+```
+
+Next, issue a rolling update to your Dapr enabled deployments. When the pods restart, the new sidecar version will be picked up.
+
+```
+kubectl rollout restart deploy/<DEPLOYMENT-NAME>
+```
+
+All done!
+
+#### Starting fresh
+
+If you previously installed Dapr on your Kubernetes cluster using the Dapr CLI, run:
+
+*Note: Make sure you're uninstalling with your existing CLI*
+
+```bash
+dapr uninstall --kubernetes
+```
+
+It's fine to ignore any errors that might show up.
+
+If you previously installed Dapr using __Helm 2.X__:
+
+```bash
+helm del --purge dapr
+```
+
+If you previously installed Dapr using __Helm 3.X__:
+
+```bash
+helm uninstall dapr -n dapr-system
+```
+
+Update the Dapr repo:
+
+```bash
+helm repo update
+```
+
+If you installed Dapr with Helm to a namespace other than `dapr-system`, modify the uninstall command above to account for that.
+
+You can now follow [these](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#using-helm-advanced) instructions on how to install Dapr using __Helm 3__.
+
+Alternatively, you can use the CLI:
+
+```
+dapr init --kubernetes
+```
+
+#### Post installation
+
+Verify the control plane pods are running and are healthy:
+
+```
+dapr status -k
+```
+
+After Dapr 0.11.0 has been installed, perform a rolling restart for your deployments to pick up the new version of the sidecar.
+This can be done with:
+
+```
+kubectl rollout restart deploy/<deployment-name>
+```
+
+## Breaking Changes
+
+### Dapr Runtime
+
+#### New ClusterRole for dapr-operator
```

Comment:
Why is this a breaking change?

Reply:
```
Error: UPGRADE FAILED: cannot patch "dapr-operator" with kind ClusterRoleBinding: ClusterRoleBinding.rbac.authorization.k8s.io "dapr-operator" is invalid: roleRef: Invalid value: rbac.RoleRef{APIGroup:"rbac.authorization.k8s.io", Kind:"ClusterRole", Name:"dapr-operator-admin"}: cannot change roleRef
```
@yaron2 
Getting this error while following the steps defined in https://github.com/dapr/docs/tree/master/howto/deploy-k8s-prod#upgrading-dapr-with-helm

Reply:
Try adding `--force` to the upgrade command?